src/window/editor/scenario_event_details.c

```diff
@@ -259,7 +262,8 @@ static void update_visible_conditions_and_actions(void)
     data.actions.active = 0;
     if (data.actions.available) {
         scenario_action_t *action;
-        array_foreach(data.event->actions, action) {
+        array_foreach(data.event->actions, action)
+        {
```

Comment:
```suggestion
        array_foreach(data.event->actions, action) {
```

The auto formatting is annoying!